Made paragraph in README more readable. (#505)
pypa/twine: README.rst

security and testability.

The biggest reason to use ``twine`` is that it securely authenticates
you to `PyPI`_ over HTTPS using a verified connection, regardless of
the underlying Python version. Meanwhile, ``python setup.py upload``
will only work correctly and securely if your build system, Python

> version and the underlying operating system are configured properly.

Reviewer: ```suggestion
version, and underlying operating system are configured properly.
```

> Secondly, ``twine`` allows you to precreate your distribution files. ``python

Reviewer 2: ```suggestion
Secondly, ``twine`` encourages you to build your distribution files before uploading them. ``python
```
